tlx/logger/core.hpp

```diff
@@ -231,7 +231,8 @@ template <typename AnyType>
 class LoggerFormatter<AnyType>
 {
 public:
-    static void print(std::ostream& os, const AnyType& t) {
+    static void print(std::ostream& os, const AnyType& t, const bool addSpace) {
+	(void)addSpace;
```

Comment:
why not just `const bool /* addSpace */` and save the cast to void?

Reply:
err, this won't fly. extra bool for just one type? nope.

Reply:
it's for every container type, not just one.